### ISSUE TITLE
Enrich the label functionality for Timesheet

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#634]
### TLDR ###
Enrich the label functionality by adding possibility to show certain messages to the user.

### Discussion ###
Could we allow for some extra fields e.g. `infoMessage`, `warningMessage` (and potentially `errorMessage`) on the `LabelObject` entity?

**An example:**

You set `warningMessage` for the label `external-timereg` to:

> You have **n** events connected to the label external-timereg. Make sure you have registered these hours in the external time registry.

If the user have some events connected to the `external-timereg` label, when confirming the hours he/she will have a confirmation dialog showing up with this message. 

E.g.:

![image](https://user-images.githubusercontent.com/7606007/98340606-e430d000-200d-11eb-9d27-ea0fe27ebfb5.png)

_This would show as a warning message obviously, and it would support several message if applicable._

### Related issues ###
No related issues.
